### PR TITLE
Xiaomi Zigbee - Pressure, Motion and illuminance improvements

### PR DESCRIPTION
**Pull request by Beejeedeb**
WSDCGQ11LM (square) changed the pressure part; unit is now hPa with a minimum of 800 and maximum of 1100 and I used ParseNumericAttr for the parseValueFromAttr. Under normal circumstances the atmospheric pressure falls between 800 and 1100 hPa. I have compared the numbers from the pressure sensor with a nearby weather station and the numbers are comparable.

RTCGQ11LM changed the illuminance part; type is now levelproperty, lux has now a maximum of 1500 and used ParseNumericAttr for the parseValueFromAttr. it is still not very accurate, but it is enough to detect changes in light to act accordingly to it.

Change label from the motion sensors properties from Occupied to Motion.

Relates to issues:
https://github.com/mozilla-iot/zigbee-adapter/issues/138 
https://github.com/mozilla-iot/zigbee-adapter/issues/139

### CODE REVIEW

**Comment by Beejeedeb** [comment]
Isn't the name field used/found when you discover the device? Discover name
is zb-number-motion.

I think (not sure after an update) that you always must rediscover the
sensors otherwise the changes will not be visible. So rules will be broken?

File: zb-xiaomi.js [hunk 1]
```diff
@@ -188,7 +189,7 @@ const MODEL_IDS = {
         clusterId: CLUSTER_ID.ILLUMINANCE_MEASUREMENT,
         attr: 'measuredValue',
         value: 0,
-        parseValueFromAttr: 'parseIlluminanceMeasurementAttr',
+        parseValueFromAttr: 'parseNumericAttr',
       },
```
Comment (dhylands) [inline zb-xiaomi.js]: It isn't clear why you changed this. The Spec says that the number presented in measureValue isn't lux but rather a forumla which parseIllumincanceMeasurementAttr does.

File: zb-xiaomi.js [hunk 2]
```diff
@@ -109,7 +109,7 @@ const MODEL_IDS = {
     },
   },
   'lumi.sensor_motion': {
-    name: 'occupancy',
+    name: 'motion',
```
[the remark this answers is not in the document]
Comment (dhylands) [reply zb-xiaomi.js]: You are correct. I was mixing up the property name and the sensor name.  So leave this as motion.

File: zb-xiaomi.js (hunk 1 above)
Comment (Beejeedeb) [reply zb-xiaomi.js]: if you use parseIlluminanceMeasurementAttr you get an strange number 1.xxxx something certainly not lux :-) Now you get a number which is 0 on dark and gets higher the more light you shine on it. Not sure if this is lux.

Comment (Beejeedeb) [reply zb-xiaomi.js]: zigbee: zb-00158d000302cf27-motion property: illuminance profileId: 0104 endpoint: 1 clusterId: 0400 report value: 126

In the logging you can see that last value in this case 126 varies with light intensity.

Comment (dhylands) [reply zb-xiaomi.js]: OK - I'll leave it like it is for now. I ordered an RTCGQ11LM and when it arrives I'll compare it other devices and see it the numbers make any sense.

Comment (Beejeedeb) [reply zb-xiaomi.js]: Flashlight:
zigbee: zb-00158d000302cf27-motion property: illuminance profileId: 0104 endpoint: 1 clusterId: 0400 report value: 713

**Comment by dhylands** [comment]
These changes are now included in the 0.7.12 release of the zigbee adapter

**Pull request merged**